<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_Dashboard</class>
 <widget class="QMainWindow" name="UI_Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">.QWidget{
background-color: rgb(0, 33, 48);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="search">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
 border: 2px solid gray;
 border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>search...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QWidget" name="profileWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="IconProfile">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../Asset/Icon/Admin1.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="NamaAkun">
           <property name="styleSheet">
            <string notr="true">font-size: 20px; font-weight: bold; text-align: center; color: #ffffff;</string>
           </property>
           <property name="text">
            <string>Ryan</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Role">
           <property name="styleSheet">
            <string notr="true">font-size: 20px; font-weight: bold; text-align: center; color: #000000;
background-color: rgb(255, 255, 0);</string>
           </property>
           <property name="text">
            <string>Administrator</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2" rowspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;</string>
      </property>
      <property name="currentIndex">
       <number>7</number>
      </property>
      <widget class="QWidget" name="page1"/>
      <widget class="QWidget" name="page2"/>
      <widget class="QWidget" name="page3"/>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page4"/>
      <widget class="QWidget" name="page5"/>
      <widget class="QWidget" name="page6"/>
      <widget class="QWidget" name="page7"/>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QWidget" name="HeaderLabel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="icon_label">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../Asset/Icon/Buku.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="title_label">
         <property name="styleSheet">
          <string notr="true">font-size: 26px; font-weight: bold; text-align: center; color: #ffffff;</string>
         </property>
         <property name="text">
          <string>Perpustakaan Digital
      Kelompok 2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>298</width>
         <height>518</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 33, 48);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="Menu1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px; font-weight: bold; text-align: center; color: #555555;
background-color: rgb(0, 24, 35);</string>
          </property>
          <property name="text">
           <string>Main Navigation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="home">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border:tranparanted
</string>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Asset/Icon/home.png</normaloff>../Asset/Icon/home.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Koleksi">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
          </property>
          <property name="text">
           <string>Koleksi</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Asset/Icon/koleksi.png</normaloff>../Asset/Icon/koleksi.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="KoleksiSubMenu" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>90</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="Fiksi">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
             </property>
             <property name="text">
              <string>Fiksi</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Asset/Icon/book.png</normaloff>../Asset/Icon/book.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="NonFiksi">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
             </property>
             <property name="text">
              <string>Non Fiksi</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Asset/Icon/book.png</normaloff>../Asset/Icon/book.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Data">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
          </property>
          <property name="text">
           <string>Kelola Data</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Asset/Icon/open-folder.png</normaloff>../Asset/Icon/open-folder.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="DataSubMenu" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>90</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="DataBuku">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
             </property>
             <property name="text">
              <string>Data Buku</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Asset/Icon/book.png</normaloff>../Asset/Icon/book.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DataAnggota">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
             </property>
             <property name="text">
              <string>Data Anggota</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Asset/Icon/user.png</normaloff>../Asset/Icon/user.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LogData">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
          </property>
          <property name="text">
           <string>Log Data</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Asset/Icon/log-data.png</normaloff>../Asset/Icon/log-data.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Menu2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px; font-weight: bold; text-align: center; color: #555555;
background-color: rgb(0, 24, 35);</string>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Profile">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
          </property>
          <property name="text">
           <string>Profile</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Asset/Icon/user.png</normaloff>../Asset/Icon/user.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Logout">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px; font-weight: bold; text-align: left; color: #ffffff;padding-left: 20px;border: none;
</string>
          </property>
          <property name="text">
           <string>Logout</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Asset/Icon/logout.png</normaloff>../Asset/Icon/logout.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
